src/utils/time.utils.ts

```diff
@@ -5,3 +5,7 @@ const HOUR_IN_MS = 60 * MINUTE_IN_MS;
 export default function isInTheLastHour(date: Date): boolean {
   return Date.now() - date.getTime() < HOUR_IN_MS;
 }
+
+export function isInTheLastHours(date: Date, hours: number): boolean {
```

Comment:
aren't we keep track of times in the binary? why do we need this in the plugin as well?

Reply:
this is just for not displaying the "lost connection" notification every time we get a failure response (it may cause a lot of notifications).